package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wootric/react-native-wootric",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Hey @diegoserranoa what's with bumping only the version of the library itself? 🤔